src/lib/trakeva.ml

```diff
@@ -1,5 +1,5 @@
 (**************************************************************************)
-(*  Copyright 2014, Sebastien Mondet <seb@mondet.org>                     *)
+(*  Copyright 2015, Sebastien Mondet <seb@mondet.org>                     *)
```

Comment:
Read up on this. And it turns out that only the _original_ year is the most important. I think that updating the year is essentially just good practice.